### DISCUSSION TITLE
[#1064977] A way to create "procedural" walls (and maybe other structures) from a single segment

### DISCUSSION THREAD

**Discussion by CraftyAdventurer** [Mojang/minecraft-editor#91] [Feedback]
I think a way to create a simple portion of a wall and then being able to use it as a "pattern template" to create a longer wall would be really cool. Each segment would automatically connect to the previous one, rotating accordingly (if necessary) and the bottom part of the wall would automatically connect to the ground, independently from the height. This would speed up the process of creating detailed walls and maybe other structures. [Here's a link to a video](https://youtu.be/9nMGTju5xtM?t=60) showcasing a similar feature of the upcoming video game Hytale.

**Comment by minecraftchloe** [comment]
This could potentially be a candidate for an extension, leveraging the scripting engine within the Editor!
